src/sparsezoo/api/utils.py

```diff
@@ -32,7 +32,21 @@ def to_snake_case(string: str):
 
 
 def map_keys(
-    dictionary: Dict[str, str], mapper: Callable[[str], str]
+    dictionary: Dict[str, Any], mapper: Callable[[str], str]
 ) -> Dict[str, str]:
-    """Given a dictionary, update its key to a given mapper callable"""
-    return {mapper(key): value for key, value in dictionary.items()}
+    """
+    Given a dictionary, update its keys to a given mapper callable.
+        If the value of the dict is a List of Dict or Dict of Dict,
+        recursively map its keys
```

Comment:
Is this indentation typical?
```suggestion
    Given a dictionary, update its keys to a given mapper callable.

    If the value of the dict is a List of Dict or Dict of Dict, recursively map
    its keys
```